components/blitz/test/omero/gateway/model/ShapeSettingsTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ *------------------------------------------------------------------------------
+ *  Copyright (C) 2017 University of Dundee & Open Microscopy Environment.
+ *  All rights reserved.
+ *
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ *------------------------------------------------------------------------------
+ */
+package omero.gateway.model;
+
+import java.awt.Color;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import omero.model.Shape;
+
+
+/**
+ * Tests the encoding/decoding of some properties of ShapeSettingsData
+ * @author Jean-Marie Burel &nbsp;&nbsp;&nbsp;&nbsp;
+ * <a href="mailto:j.burel@dundee.ac.uk">j.burel@dundee.ac.uk</a>
+ * @since 5.3
+ */
+@Test(groups = "unit")
+public class ShapeSettingsTest {
+
+    @DataProvider(name = "colors")
+    public static Object[][] colors() {
+       return new Integer[][] {
+               {Integer.valueOf(255), Integer.valueOf(0), Integer.valueOf(0),
```

Comment:
opted for that way of declaring the value since it will generate warning in Java 9